### PR TITLE
Allow usage of --config param

### PR DESCRIPTION
**Pull request by stelmakh**
As requested in #1509, let's allow user to specify a path to their brunch-config file.

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -59,6 +60,7 @@ program.command('watch [path]')
   .option('-P, --port [port]', 'if `server` was given, listen on this port')
   .option('-d, --debug [pattern]', 'print verbose debug output to stdout')
   .option('-j, --jobs [num]', 'parallelize the build')
+  .option('-c, --config [path]', 'specify a path to brunch config file')
```
Comment (shvaikalesh) [inline lib/cli.js]: and here. Otherwise LGTM, thanks for the PR.

**Comment by yavorsky** [comment]
Why this was removed previously?

File: lib/cli.js [hunk 2]
```diff
@@ -48,6 +48,7 @@ program.command('build [path]')
   .option('-p, --production', 'same as `--env production`')
   .option('-d, --debug [pattern]', 'print verbose debug output to stdout')
   .option('-j, --jobs [num]', 'parallelize the build')
+  .option('-c, --config [path]', 'specify a path to brunch config file')
```
Comment (stelmakh) [inline lib/cli.js]: Sure thing

**Comment by stelmakh** [comment]
As was said in the deprecation message, it has been replaced with -e option that specifies the environment to use. Probably the --config option has been used mainly for different environments config files rather than to specify the path to config itself.

**Pull request merged**